### DISCUSSION TITLE
Feat: Add Balance control to support different Left and Right channel volume

### DISCUSSION THREAD

**Discussion by Robert-Nelson** [jcorporation/myMPD#1194] [Ideas]

**Comment by jcorporation** [comment]
Does MPD support this?

**Reply by john-** [reply]
Not sure about MPD but HTML 5 Audio doesn't support it (I assume that is what myMPD uses for local playback).

In my personal experience balance changes would be done rarely so could not justify having it on the front end in a UI.  Actually, I never have changed the balance except in my car when there are no passengers :)

@Robert-Nelson, how often are you changing the balance that it is inconvenient to use the underlying audio hardware/driver controls?

**Reply by Robert-Nelson** [reply]
I have a piano that uses the PianoDisc system. Content for this system has the MIDI data for the piano encoded on one channel and the accompaniment  (vocals, other instruments) on the other channel. Balance is used to adjust the relative volume between the piano and the accompaniment. So it is a requirement for me and the lack of this precludes my use of this client and possibly mpd entirely.

I'm not concerned about local playback, rather I'm interested in using mympd as a controller for mpd outputing to a different mpd audio_output rather than an http stream.

**Reply by jcorporation** [reply]
First check if MPD supports this. If MPD supports this we can discuss how/if can be implemented in myMPD. I am not aware of any balance control in MPD.
